docs/framework/additional-apis/system-windows-forms/updatestylescore-method.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Learn more about the UpdateStylesCore method
+title: UpdateStylesCore method
+ms.date: 11/10/2021
+ms.technology: dotnet-winforms
+api_name:
+  - System.Windows.Forms.Control.UpdateStylesCore
```

Comment:
The API showed up as System.Windows.Forms.Form.UpdateStylesCore in APIScan. But it seems to be defined in the Control class, so that's what I put here. Not sure if this is 100% correct or if it will satisfy APIScan.

Reply:
```suggestion
  - System.Windows.Forms.Control.UpdateStylesCore
  - System.Windows.Forms.Form.UpdateStylesCore
```